src/test/java/pl/edu/icm/rds/extension/citation/RdsCitationFormatsConverterTest.java

```diff
@@ -1,19 +1,21 @@
 package pl.edu.icm.rds.extension.citation;
 
 import edu.harvard.iq.dataverse.citation.CitationData;
-import edu.harvard.iq.dataverse.citation.CitationLocalizedConstantsService;
 import edu.harvard.iq.dataverse.persistence.GlobalId;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 import java.util.Arrays;
+import java.util.Locale;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 
 class RdsCitationFormatsConverterTest {
 
-    private RdsCitationFormatsConverter converter = new RdsCitationFormatsConverter(new CitationLocalizedConstantsService());
+    private RdsCitationFormatsConverter converter = new RdsCitationFormatsConverter();
+
+    private static final Locale TEST_LOCALE = Locale.ENGLISH;
```

Comment:
maybe we could add a test for different locale than english?

Reply:
Ok, but let's move that test issue to next task (i.e. the changes I'm currently doing), as test data will have to be rewritten, so it's better to do it once.